src/Fable.Compiler/Library.fs

```diff
@@ -260,8 +260,23 @@ module CodeServices =
         : Async<CompileResult>
         =
         async {
+            let signatureFile =
+                crackerResponse.ProjectOptions.SourceFiles
+                |> Array.tryFind (fun f -> f = String.Concat(currentFile, "i"))
+
             let! dependentFiles =
-                checker.GetDependentFiles(currentFile, crackerResponse.ProjectOptions.SourceFiles, sourceReader)
+                match signatureFile with
+                | None ->
+                    checker.GetDependentFiles(currentFile, crackerResponse.ProjectOptions.SourceFiles, sourceReader)
+                | Some signatureFile ->
+                    checker.GetDependentFiles(signatureFile, crackerResponse.ProjectOptions.SourceFiles, sourceReader)
+
+            Log.info (
+                sprintf
+                    "Dependent files for %s are:\n%s"
+                    (Option.defaultValue currentFile signatureFile)
+                    (String.concat "\n" dependentFiles)
+            )
```

Comment:
I think we should remove this log as we don't have yet a debug mode for the log and it will "pollute" the already really verbose `-- verbose` mode.

And I think this log is mostly for debug purpose.

What do you think?

Reply:
I added this log for a reason, it is incredibly useful and this is not called by `Fable.Cli`.
It happens exclusively in `Fable.Compiler` API endpoint. Users will never see this and I want to have this log.

Reply:
Ah ok no problem, I thought it was going to go to the end user too.